esmvaltool/diag_scripts/monitor/multi_datasets.py

```diff
@@ -574,9 +626,8 @@ def _get_cbar_label(self, plot_type, dataset, bias=False):
         cbar_label = self._fill_facet_placeholders(cbar_label, dataset, descr)
         return cbar_label
 
-    def _get_gridline_kwargs(self):
+    def _get_gridline_kwargs(self, plot_type='map'):
```

Comment:
```suggestion
    def _get_gridline_kwargs(self, plot_type):
```

I think it's better to make that non-optional, a default value `map` is not really reasonable